docs_user/modules/proc_using-new-subnet-ranges.adoc

```diff
@@ -46,59 +26,53 @@ network_config:
-new and old parts of the deployment.
-
-Once done, run `openstack overcloud node provision` to apply the new configuration.
-
-Note that network configuration changes are not applied by default to avoid
-risk of network disruption. You will have to enforce the changes by setting the
++
+Repeat this step for other networks that need to use different subnets for the new and old parts of the deployment.
+
+. Apply the new configuration to every adopted node:
++
+----
+$ openstack overcloud node provision
+--stack <stack>
+--network_config
+----
++
+[NOTE]
+Network configuration changes are not applied by default to avoid
+the risk of network disruption. You must enforce the changes by setting the
 `StandaloneNetworkConfigUpdate: true` in the {OpenStackPreviousInstaller} configuration files.
 
-Once `openstack overcloud node provision` is complete, you should see new link local routes to the
-new subnet on each node. For example,
-
-```bash
+. Confirm that there are new link local routes to the new subnet on each node. For example:
++
+----
 # ip route | grep 172
 172.31.0.0/24 via 192.168.122.100 dev br-ctlplane
-```
-
----
-
-The next step is to configure similar routes for the old subnet for control plane services attached to the networks. This is done by adding `routes` entries to
-`NodeNetworkConfigurationPolicy` CRs for each network. For example,
+----
 
-```yaml
+. To configure similar routes for the old subnet for the control plane services that are attached to the networks, add `routes` entries to the `NodeNetworkConfigurationPolicy` CRs for each network. For example:
++
+----
       - destination: 192.168.122.0/24 <1>
         next-hop-interface: ospbr <2>
-```
-<1> The isolated network's original subnet on the data plane.
-<2> The {OpenShiftShort} worker network interface that corresponds to the isolated network on the data plane.
-
-Once applied, you should eventually see the following route added to your {rhocp_long} nodes.
-
-```bash
+----
+<1> The original subnet of the isolated network on the data plane.
+<2> The {rhocp_long} worker network interface that corresponds to the isolated network on the data plane.
++
+As a result, the following route is added to your {OpenShiftShort} nodes:
++
+----
 # ip route | grep 192
 192.168.122.0/24 dev ospbr proto static scope link
-```
+----
++
+At this point, you can ping the adopted nodes from the {OpenShiftShort} nodes by using the old subnet IP addresses.
 
----
-
-At this point, you should be able to ping the adopted nodes from {OpenShiftShort} nodes
-using their old subnet addresses; and vice versa.
-
----
-
-
-Finally, during the data plane adoption, you will have to take care of several aspects:
-
-- in network_config, add link local routes to the new subnets, for example:
-
-```yaml
+. During the data plane adoption, in the `network_config` section of the `NetworkAttachmentDefinition` CR, add link local routes to the new subnets. For example:
```

Comment:
This is not referring to `NetworkAttachmentDefinition`. The configuration will go into `OpenStackDataPlaneNodeSet` CR.